### PR TITLE
IDENTITY-3501 improvements

### PR DESCRIPTION
**Pull request by cdwijayarathna**

### CODE REVIEW

File: components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/bean/WorkFlowRequest.java [hunk 1]
```diff
@@ -73,10 +74,12 @@ public String getRequestParameterAsString() {
 
         String requestParametersString = "{";
         for (int i = 0; i < requestParameters.size(); i++) {
-            requestParametersString = requestParametersString + requestParameters.get(i).getName() + " : " +
-                    requestParameters.get(i).getValue();
-            if (i != requestParameters.size() - 1) {
-                requestParametersString = requestParametersString + ", \n";
+            if(!requestParameters.get(i).getName().equals(CREDENTIAL)) {
```
Comment (pulasthi7) [inline components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/bean/WorkFlowRequest.java]: flip equals()

File: components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/dao/WorkflowRequestDAO.java [hunk 2]
```diff
@@ -254,18 +258,19 @@ public void updateStatusOfRequest(String requestId, String newState) throws Inte
...
+    * @param endTime upper limit of date range to filter
+    * @param timeCategory filter by created time or last updated time ?
+    * @param tenantId tenant id of currently logged in user
      * @return
      * @throws InternalWorkflowException
      */
     public WorkflowRequestDTO[] getRequestsOfUserFilteredByTime(String userName, Timestamp beginTime, Timestamp
-            endTime, String timeCategory) throws
+            endTime, String timeCategory, int tenantId) throws
             InternalWorkflowException {
```
Comment (pulasthi7) [inline components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/dao/WorkflowRequestDAO.java]: can be appended to previous line

Comment (cdwijayarathna) [inline components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/dao/WorkflowRequestDAO.java]: Fixed with https://github.com/wso2/carbon-identity/pull/1340

File: components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/bean/WorkFlowRequest.java (hunk 1 above)
Comment (cdwijayarathna) [inline components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt/src/main/java/org/wso2/carbon/identity/workflow/mgt/bean/WorkFlowRequest.java]: Fixed with https://github.com/wso2/carbon-identity/pull/1675

**Pull request merged**